fladle-plugin/src/test/java/com/osacky/flank/gradle/integration/FulladlePluginIntegrationTest.kt

```diff
@@ -241,4 +241,370 @@ class FulladlePluginIntegrationTest {
       """.trimIndent()
     )
   }
+
+  @Test
+  fun `test root level module overrides with fulladleModuleConfig`() {
+    val appFixture = "android-project"
+    testProjectRoot.newFile("settings.gradle").writeText(
+      """
+        include '$appFixture'
+
+        dependencyResolutionManagement {
+          repositories {
+            mavenCentral()
+            google()
+          }
+        }
+      """.trimIndent()
+    )
+    testProjectRoot.setupFixture(appFixture)
+
+    writeBuildGradle(
+      """
+        buildscript {
+            repositories {
+                google()
+                jcenter()
+            }
+
+            dependencies {
+                classpath 'com.android.tools.build:gradle:4.2.1'
```

Comment:
can we get this version from a local variable to avoid repeating and making upgrades easier?